CompanyEmployees.Core.Domain/Entities/Employee.cs

```diff
@@ -20,7 +20,10 @@ public class Employee
         public string? Position { get; set; }
 
         [ForeignKey(nameof(Company))]
```

Comment:
This field appears to be a temporary placeholder with a TODO comment. Leaving such fields in the domain model violates Clean Architecture principles and can lead to confusion or bugs if accidentally used. Remove it before merging.

```suggestion
// Removed temporary field: public string salary;
```